CHANGES.txt

```diff
@@ -18,6 +18,8 @@ Deprecations:
 
 Changes:
 
+- The Python file VSI plugin is now created and installed at the same time
```

Comment:
Since this feature has not been released, adding this note in the changes may not be necessary.